app/views/budgets/index.html.erb

```diff
@@ -6,22 +6,53 @@
 
 <% content_for :sidebar do %>
   <div flex layout-padding>
-    <br><br>
-    <%= render 'form' %>
+    <h3>Create Bugget</h3>
+
+<%= form_for(@budget) do |f| %>
+
+  <% if @budget.errors.any? %>
+      <%= render 'form' %> 
+  <% end %>
+
+  <%= f.label "Budget Number / short name" %>
+  <%= f.text_field :name, placeholder: "Your budget number/name"%>
+
+  <%= f.label :description %>
+  <%= f.text_field :description, placeholder: "Budget description" %>
+
+  <%= f.label "Contact" %>
+  <%= f.text_field :contact, placeholder: "Contact name" %>
+
+  <%= f.label "Contact's email" %>
+  <%= f.text_field :email, placeholder: "Contact email"%>
+
+  <%= f.label "Contact's phone" %>
+  <%= f.text_field :phone, placeholder: "Contact phone"%>
+
+  <div class="actions">
+    <br />
+    <% if @budget.id %>
+      <%= f.submit 'Save', class: 'md-button md-raised md-primary' %>
+    <% else %>
+      <%= f.submit 'New Budget', class: 'md-button md-raised md-primary' %>
+    <% end %>
+       
+<% end %>
+
+</div>
+
   </div>
 <% end %>
 
 <% content_for :main do %>
 
-  <div flex layout-padding>
-
     <table class='table'>
       <tr>
-        <th>Name</th>
-        <th>Description</th>
-        <th>Contact</th>  
-        <th>Users</th>      
-        <th></th>
+        <th style="width: 10%">Name</th>
+        <th style="width: 30%">Description</th>
+        <th style="width: 10%">Contact</th>  
+        <th style="width: 40%">Users</th>      
+        <th style="width: 5%"></th>
       </tr>
 
       <% @budgets.each do |budget| %>
```

Comment:
let's use the alphabetical pagination here - sort on budget name

Reply:
I added the alphabetical pagination